lib/cli/report-issue.js

```diff
@@ -0,0 +1,40 @@
+
+
+var opener = require('opener');
+var analytics = require('./analytics');
```

Comment:
Go for it, sometimes it is easy to miss this stuff as the code continues changing.